experimental_official_dockerfiles/master/bashrc

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Couple things about the bashrc:

- Hard to make sure the python environment is correctly set up when e.g. running a non-interactive command. 
- Should align permissions stuff with a best practices doc for using cache and/or a mounted copy of the source code

Reply:
Do you think that also the derived official Vsocde/Codespace Devcontainer image bash script will go to invalidate the cache https://github.com/microsoft/vscode-dev-containers/blob/master/repository-containers/github.com/tensorflow/tensorflow/.devcontainer/Dockerfile#L17 ?

Reply:
Also, we already had a ticket for the user permission https://github.com/tensorflow/build/issues/8